src/PrivateDns/PrivateDns/Zones/RemoveAzurePrivateDnsZone.cs

```diff
@@ -57,28 +68,27 @@ public override void ExecuteCmdlet()
             if (!string.IsNullOrEmpty(this.Name) && this.Name.EndsWith("."))
             {
                 this.Name = this.Name.TrimEnd('.');
-                this.WriteWarning($"Modifying zone name to remove terminating '.'.  Zone name used is \"{this.Name}\".");
+                this.WriteWarning($"Modifying Private DNS zone name to remove terminating '.'.  Zone name used is \"{this.Name}\".");
             }
 
             var zoneToDelete = (this.ParameterSetName != "Object")
```

Comment:
Use constants for param sets instead of defining them in-line.

Reply:
done.